src/_assets/css/_sidebar.scss

```diff
@@ -52,22 +54,20 @@
     }
   }
 
-  // Second nav level
-  > .nav > .nav > .nav-item {
+  // Second-level nav item
+  > .nav > .nav-item > .nav > .nav-item {
     font-size: $font-size-sm;
 
     .nav-link {
```

Comment:
To be consistent, we should probably add `> ` here as well since you added it to the top level one.

Reply:
Done here a below